rack-protection/lib/rack/protection/authenticity_token.rb

```diff
@@ -12,11 +12,74 @@ module Protection
     # Only accepts unsafe HTTP requests if a given access token matches the token
     # included in the session.
     #
-    # Compatible with rack-csrf.
+    # Compatible with the {rack-csrf}[https://rubygems.org/gems/rack_csrf] gem.
     #
-    # Options:
+    # The middleware only accepts unsafe HTTP requests (exclude <tt>GET</tt>,
+    # <tt>HEAD</tt>, <tt>OPTIONS</tt>, <tt>TRACE</tt>) if a given access token
+    # matches the token included in the session.
```

Comment:
This might be a little confusing here. Maybe say (requests other than <tt>GET</tt, ..)